### PR TITLE
Fixes Issue #2272

### PR DESCRIPTION
**Pull request by jatinAroraGit**
Hey there,
This PR adds a rating feature for mangas in a user's library.

Other feature requested was adding support for marking sources favorite but it would require a major overhaul of the classes and DB  and how getters/setters will work so it would be better to create a separate issue for it. (there is one already filed issue https://github.com/inorichi/tachiyomi/issues/2322 )

The global search feature already exists in the latest version of application.

Let me know if there are any issues with the PR.

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryListHolder.kt [hunk 1]
```diff
@@ -5,7 +5,14 @@ import com.bumptech.glide.load.engine.DiskCacheStrategy
...
 import eu.kanade.tachiyomi.data.glide.GlideApp
 import eu.kanade.tachiyomi.source.LocalSource
+import kotlinx.android.synthetic.main.catalogue_grid_item.*
 import kotlinx.android.synthetic.main.catalogue_list_item.*
+import kotlinx.android.synthetic.main.catalogue_list_item.download_text
+import kotlinx.android.synthetic.main.catalogue_list_item.local_text
+import kotlinx.android.synthetic.main.catalogue_list_item.rating
+import kotlinx.android.synthetic.main.catalogue_list_item.thumbnail
+import kotlinx.android.synthetic.main.catalogue_list_item.title
+import kotlinx.android.synthetic.main.catalogue_list_item.unread_text
```
Comment (NotTsunami) [inline app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryListHolder.kt]: The already listed `import kotlinx.android.synthetic.main.catalogue_list_item.*` should cover these, correct?

File: app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryGridHolder.kt [hunk 2]
```diff
@@ -6,6 +6,13 @@ import eu.davidea.flexibleadapter.FlexibleAdapter
 import eu.kanade.tachiyomi.data.glide.GlideApp
 import eu.kanade.tachiyomi.source.LocalSource
 import kotlinx.android.synthetic.main.catalogue_grid_item.*
+import kotlinx.android.synthetic.main.catalogue_grid_item.download_text
+import kotlinx.android.synthetic.main.catalogue_grid_item.local_text
+import kotlinx.android.synthetic.main.catalogue_grid_item.rating
+import kotlinx.android.synthetic.main.catalogue_grid_item.thumbnail
+import kotlinx.android.synthetic.main.catalogue_grid_item.title
+import kotlinx.android.synthetic.main.catalogue_grid_item.unread_text
```
Comment (NotTsunami) [inline app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryGridHolder.kt]: Same comment here

File: app/src/main/java/eu/kanade/tachiyomi/ui/manga/info/MangaInfoController.kt [hunk 3]
```diff
@@ -91,6 +93,9 @@ class MangaInfoController : NucleusController<MangaInfoPresenter>(),
         // Set onLongClickListener to manage categories when FAB is clicked.
         fab_favorite.longClicks().subscribeUntilDestroy{ onFabLongClick() }
 
+        //Set onclickListener to open alert dialog for rating the manga.
```
Comment (NotTsunami) [inline app/src/main/java/eu/kanade/tachiyomi/ui/manga/info/MangaInfoController.kt]: Nit: Space after //

**Comment by arkon** [comment]
As mentioned in the issue, not a fan of this feature.

**Pull request closed without merging**